sqlalchemy_history/unit_of_work.py

```diff
@@ -113,6 +113,18 @@ def create_transaction(self, session):
         if not self.version_session:
             self.version_session = sa.orm.session.Session(bind=session.connection())
         self.version_session.add(self.current_transaction)
+
+        # Fetch the last-id in Transaction table
+        # We are manually setting the transaction ID - to guarantee that the transaction ID is always
+        # increasing. In some cases like Oracle RAC - the order of the IDs generated are not sequential.
+        # Ref: https://stackoverflow.com/questions/57620096
+        # So, here we ourselves are setting the transaction IDs using table locks
+
+        # FIXME: There is a small race condition here when 2 APIs could write concurrently between the
+        #        last_transaction_id fetching and FLUSH
+        last_transaction_id = self.version_session.query(sa.func.max(Transaction.id)).scalar()
+        self.current_transaction.id = last_transaction_id
```

Comment:
should we not be doing `last_txn_id + 1` ?

Reply:
Yes -_- will fix